src/pip/_internal/utils/appdirs.py

```diff
@@ -190,6 +190,11 @@ def site_config_dirs(appname):
         # always look in /etc directly as well
         pathlist.append('/etc')
 
+    # allow configuration in sys.prefix without any appname
+    # but put it first so that it is not preferred for "pip config set"
+    # see <https://github.com/pypa/pip/issues/5060>
+    pathlist.insert(0, sys.prefix)
```

Comment:
Uhm... Please don't modify the appdirs file. #6040

Reply:
As I mentioned to Paul above, there is exactly one use of the forked appdirs (here) and one use of the vendored appdirs (in vendored pkg_resources).

And #6040 says nothing about the `site_config_dirs` function from forked appdirs, which is the only function used from it.

So it isn't at all obvious which is the "right" file, except that switching to the vendored one without modifying it to match the forked one would be a serious regression.

Reply:
Yea, I don't like the appdirs situation. I want to switch us to the vendored version but there's like, Py2-only fixes which is an area I am do not want to deal with for now. upstream isn't exactly responsive either.

I'm almost sure that the right thing to do now is to patch our vendored appdirs, carrying patches for all of the "fixes" in pip's forked version and drop our forked version but I just haven't come around to it yet.

Reply:
Basically, the reason I'm asking to not modify the forked appdirs is that it'll make it harder to switch to whatever is vendored (and patched).